app/Models/Dinnerform.php

```diff
@@ -57,4 +60,48 @@ public function hasExpired()
     {
         return $this->end->addHours(1)->isPast();
     }
+
+    /** @return HasMany|DinnerformOrderline[] */
+    public function orderLines()
+    {
+        return $this->hasMany('Proto\Models\DinnerformOrderline');
+    }
+
+    /** @return BelongsTo */
+    public function event()
+    {
+        return $this->belongsTo('Proto\Models\Event', 'event_id');
+    }
+
+    public function totalAmount() {
+        return $this->orderlines()->sum('price');
+    }
+
+    public function totalAmountwithHelperDiscount() {
```

Comment:
Helper discount should be an amount not a percentage.

Reply:
Fixed :)